### PR TITLE
added config param PGPOOL_DISABLE_LOAD_BALANCE_ON_WRITE=

### PR DESCRIPTION
**Pull request by asafo**
**Description of the change**

Adding option to configure: disable_load_balance_on_write

**Benefits**

It is very important to configure with some web apps and others

**Additional information**

Still have no validation routine

### CODE REVIEW

File: 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh [hunk 1]
```diff
@@ -289,6 +293,7 @@ pgpool_create_config() {
     local pool_hba=""
     local pool_passwd=""
     local allow_clear_text_frontend_auth="off"
+    local num_init_children=0
```
Comment (dani8art) [inline 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh:296]: This is never used again, right?

File: 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh [hunk 2]
```diff
@@ -60,6 +60,8 @@ export PGPOOL_ADMIN_USERNAME="${PGPOOL_ADMIN_USERNAME:-}"
 export PGPOOL_ENABLE_LDAP="${PGPOOL_ENABLE_LDAP:-no}"
 export PGPOOL_TIMEOUT="360"
 export PGPOOL_ENABLE_LOAD_BALANCING="${PGPOOL_ENABLE_LOAD_BALANCING:-yes}"
+export PGPOOL_DISABLE_LOAD_BALANCE_ON_WRITE="${PGPOOL_DISABLE_LOAD_BALANCE_ON_WRITE:-transaction}"
+export PGPOOL_NUM_INIT_CHILDREN="${PGPOOL_NUM_INIT_CHILDREN:-32}"
```
Comment (dani8art) [inline 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh:64]: Why do we need this?

File: 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh [hunk 3]
```diff
@@ -60,6 +60,8 @@ export PGPOOL_ADMIN_USERNAME="${PGPOOL_ADMIN_USERNAME:-}"
 export PGPOOL_ENABLE_LDAP="${PGPOOL_ENABLE_LDAP:-no}"
 export PGPOOL_TIMEOUT="360"
 export PGPOOL_ENABLE_LOAD_BALANCING="${PGPOOL_ENABLE_LOAD_BALANCING:-yes}"
+export PGPOOL_DISABLE_LOAD_BALANCE_ON_WRITE="${PGPOOL_DISABLE_LOAD_BALANCE_ON_WRITE:-transaction}"
```
Comment (dani8art) [inline 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh:63]: I would set default value to `off`, also we should check it has a valid value

File: 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh (hunk 2 above)
Comment (asafo) [reply 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh:64]: Yes, sorry, I've removed it

File: 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh (hunk 3 above)
Comment (asafo) [reply 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh:63]: The value was this before my change, and it seem's to be pgpool's default also.
From the docs:

> This setting gives the best balance in most cases and you should start from this

Comment (asafo) [reply 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh:63]: as for validating the value, we'll need another function in libvalidate, for enums

Comment (dani8art) [reply 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh:63]: we could add sth like 
```bash
## libvalidate.sh
array_contains() {
  array=${1:?array is required}
  item=${2:?item is required}

  for $i in array
  do
    if [[ $i -eq $item ]]; then
      return 1
    fi
  done

  return 0
}
## libpgpool.sh
array_contains ("off" "transaction" "trans_transaction" "always") $PGPOOL_DISABLE_LOAD_BALANCE_ON_WRITE
```

Comment (asafo) [reply 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh:63]: 10x, I'll Try to fix this  ASAP

Comment (asafo) [reply 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh:63]: I've just committed the above validation, 10x

**Pull request merged**